data/data/libvirt/variables-libvirt.tf

```diff
@@ -61,6 +61,6 @@ variable "libvirt_master_size" {
 
 variable "libvirt_dnsmasq_options" {
   type        = map(string)
-  description = "Dnsamasq options to be applied to the libvirt network"
+  description = "Dnsmasq options to be applied to the libvirt network"
```

Comment:
Can this be squashed in `libvirt: add support for dnsmasq options in the terraform files` rather than being changed in ` libvirt: support setting network dnsmasq options through the install … ` ?

Reply:
yeah that would be cleaner..i was just being lazy :) . will do.